regression-tests/test-results/apple-clang-14/mixed-captures-in-expressions-and-postconditions.cpp.output

```diff
@@ -1,7 +0,0 @@
-mixed-captures-in-expressions-and-postconditions.cpp2:12:18: error: no member named 'for_each' in namespace 'std::ranges'
-    std::ranges::for_each(vec, callback);
-    ~~~~~~~~~~~~~^
-mixed-captures-in-expressions-and-postconditions.cpp2:14:18: error: no member named 'for_each' in namespace 'std::ranges'
-    std::ranges::for_each(std::move(vec), std::move(callback));
-    ~~~~~~~~~~~~~^
-2 errors generated.
```

Comment:
That's cool!
These errors are present on `macos-latest` GitHub runners (see [here](https://github.com/jarzec/cppfront/actions/runs/5913839684/job/16038580118) under "Run regression tests - Linux and macOS version").
I am guessing this is thanks to some sort of update? What kind of update is necessary? Is it just Xcode or something more complicated? The change to `clang-version.output` seems to suggest a full OS version bump but maybe I'm mistaken.

Reply:
I guess. See <https://github.com/hsutter/cppfront/issues/262#issuecomment-1512136235>.